sentry_sdk/client.py

```diff
@@ -647,6 +660,89 @@ def capture_event(
 
         return event_id
 
+    def capture_message(
+        self,
+        message,  # type: str
+        level=None,  # type: Optional[str]
+        scope=None,  # type: Optional[Scope]
+        top_scope=None,  # type: Optional[Scope]
```

Comment:
I would not add `top_scope` here.

If you [see the RFC](https://github.com/getsentry/rfcs/blob/c4b1fb4fd2915246760156285393340af093d591/text/0122-sdk-hub-scope-merge.md?plain=1#L350-L369), the 3 scopes will be processed by `Scope.capture_event`. 

The managing of the scopes is currently done by the Hub and will be moved to Scope later. I think the client should just get one scope as before and the merging will be moved later from Hub -> Scope.